Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -653,6 +641,7 @@
   gEfiMdeModulePkgTokenSpaceGuid.PcdStatusCodeUseSerial|FALSE
   gEfiMdeModulePkgTokenSpaceGuid.PcdStatusCodeUseMemory|TRUE
   gPcAtChipsetPkgTokenSpaceGuid.PcdUartIoPortBaseAddress|0x402
+  gUefiQemuQ35PkgTokenSpaceGuid.PcdDebuggerPortUartBase|0x3F8

```

Comment:
If this is a Q35 PCD, should we default 0x3f8?

Reply:
makes sense.